datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/deser/DurationDeserializer.java

```diff
@@ -49,6 +53,11 @@
 
     public static final DurationDeserializer INSTANCE = new DurationDeserializer();
 
+    /**
+     * Since 2.12
```

Comment:
Suggest adding complete JavaDoc, see this for a template you could use: https://github.com/FasterXML/jackson-modules-java8/blob/eeb2b67de6fdc975d9bde88d32b2c1c83a18d710/datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/deser/JSR310DateTimeDeserializerBase.java#L22